Source/Core/Core/PowerPC/Jit64/Jit.cpp

```diff
@@ -554,6 +596,7 @@ const u8* Jit64::DoJit(u32 em_address, PPCAnalyst::CodeBuffer *code_buf, JitBloc
 	jit->js.numFloatingPointInst = 0;
 
 	PPCAnalyst::CodeOp *ops = code_buf->codebuffer;
+	js.blockEnd = ops[code_block.m_num_instructions - 1].address;
```

Comment:
There's also the problem that the last instruction of a block can be reordered, in theory, if the block is broken. Is there a better way to know the last address in a block?